source/_components/transmission.markdown

```diff
@@ -0,0 +1,112 @@
+---
+
+The `transmission` component allows you to monitor your downloads with [Transmission](http://www.transmissionbt.com/) from within Home Assistant and setup automation based on the information.
+
+## {% linkable_title Setup %}
+
+To use the monitoring, your transmission client needs to allow remote access. If you are running the graphical transmission client (transmission-gtk) go to **Edit** -> **Preferences** and choose the tab **Remote**. Check **Allow remote access**, enter your username and your password, and uncheck the network restriction as needed.
+
+<p class='img'>
+  <img src='{{site_root}}/images/components/transmission/transmission_perf.png' />
+</p>
+
+If everything is set up correctly, the details will show up in the frontend.
+
+<p class='img'>
+  <img src='{{site_root}}/images/components/transmission/transmission.png' />
+</p>
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+transmission:
+  host: 192.168.1.1
+```
+
+{% configuration %}
+host:
+  description: This is the IP address of your Transmission daemon, e.g., `192.168.1.1`.
+  required: true
+  type: string
+port:
+  description: The port your Transmission daemon uses.
+  required: false
+  type: integer
+  default: 9091
+name:
+  description: The name to use when displaying this Transmission instance in the frontend.
+  required: false
+  type: string
+username:
+  description: Your Transmission username, if you use authentication.
+  required: false
+  type: string
+password:
+  description: Your Transmission password, if you use authentication.
+  required: false
+  type: string
+turtle_mode:
+  description: If enabled, it creates a switch entity to control the 'Alternative Speed Limits' (aka 'Turtle mode') setting.
+  required: false
+  type: boolean
+monitored_variables:
+  description: Conditions to display in the frontend. Every one of them will act as a sensor.
+  required: false
+  type: map
+  keys:
+    current_status:
+      description: The status of your Transmission daemon.
+    download_speed:
+      description: The current download speed [MB/s].
+    upload_speed:
+      description: The current upload speed [MB/s].
+    active_torrents:
+      description: The current number of active torrents.
+    paused_torrents:
+      description: The current number of paused torrents.
+    total_torrents:
+      description: The total number of torrents present in the client.
+    started_torrents:
+      description: The current number of started torrents (downloading).
+    completed_torrents:
+      description: The current number of completed torrents (seeding)
+{% endconfiguration %}
+
+## {% linkable_title Event Automation %}
+
+The Transmission component is continuously monitoring the status of torrents in the target client. Once a torrent is started or completed, an event is triggered on the Home Assistant Bus, allows to implement any kind of automation.
```

Comment:
an event is triggered on the Home Assistant event bus, which allows to implement any kind of automation.